x-pack/plugins/endpoint/server/types.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { LoggerFactory } from 'kibana/server';
+import { EndpointConfigType } from './config';
+
+export interface EndpointAppContext {
+  logFactory: LoggerFactory;
+  config(): Promise<EndpointConfigType>;
+}
+
+export class EndpointAppConstants {
+  static ENDPOINT_INDEX_NAME = 'endpoint-agent*';
```

Comment:
perhaps the string 'endpoint-agent*' could just be in a `const` variable (as opposed to a public member in a class) outside of this `types` module.

Reply:
> perhaps the string 'endpoint-agent*' could just be in a const variable (as opposed to a public member in a class) outside of this types module.

The reason is that the bootstrap also uses this value, so I wanted to put it somewhere shared. Is that is fine.

Reply:
@nnamdifrankie was this decided somewhere - since this is going to master shouldn't this be more finalized?

Reply:
https://github.com/elastic/kibana/pull/53861#discussion_r363361744